pelican/plugins/image_process/image_process.py

```diff
@@ -722,8 +717,11 @@ def process_image(image, settings):
         or not os.path.exists(image[1])
         or os.path.getmtime(image[0]) > os.path.getmtime(image[1])
     ):
+        logger.debug("{} Processing {} -> {}".format(LOG_PREFIX, image[0], image[1]))
 
-        i = Image.open(image[0])
+        i = try_open_image(image[0])
+        if i is None:
```

Comment:
It feels like it would be better (more Python-ic) to re-raise the exemptions in `try_open_image()`, and then catch them here.

Is there other cases (than raised exemptions) that `i` could come back as `None`?